assets/airports/kpvd.json

```diff
@@ -0,0 +1,6536 @@
+{
+    "radio": {
+        "twr": "Providence Tower",
+        "app": "Providence Approach",
+        "dep": "Providence Departure"
+    },
+    "icao": "KPVD",
+    "iata": "PVD",
+    "magnetic_north": -14.5,
+    "ctr_radius": 80,
+    "ctr_ceiling": 12000,
+    "initial_alt": 4000,
+    "position": ["N41d43m44.00", "W071d25m69.00", "54ft"],
+    "rr_radius_nm": 5.0,
+    "rr_center": ["N41d43m44.00", "W071d25m69.00"],
+    "has_terrain": true,
+    "wind": {
+        "angle": 310,
+        "speed": 9
+    },
+    "arrivalRunway": "23",
```

Comment:
doesn't match the winds

Reply:
The winds were an average over a year and 5/23 is the longest runway but I shall check their rwy config later

Reply:
From what I gathered they'd use 5 and 34 in these conditions